autoload/lsp/diag.vim

```diff
@@ -71,6 +111,15 @@ def DiagsRefreshSigns(lspserver: dict<any>, bnr: number)
 				lnum: lnum,
 				name: DiagSevToSignName(diag.severity)})
 
+    if opt.lspOptions.highlightDiagInline
+      prop_add(diag.range.start.line + 1,
```

Comment:
When adding a few lines with diagnostics messages and then removing them (by doing an undo) , I see an "E966: Invalid line number" error.  It looks like by the time the diagnostics messages are processed, the lines are already removed.  I think we need to debounce the diagnostic messages.